src/api/tenantApi.js

```diff
@@ -0,0 +1,116 @@
+import axios from 'axios'
+const BASE_URL = 'https://dev.elevate-apis.shikshalokam.org'
+
+// Get all tenants
+export const getAllTenants = async () => {
+  const response = await axios.get(`${BASE_URL}/user/v1/tenant/list`, {
+    headers: {
```

Comment:
Instead of adding headers, axios, etc in every api request, figure out a generic way of doing this. For example, we need to add a new header tomorrow, and we need to add that header in all the places. Similarly, if we need to replace axios with another library, it should be just a one-place change.